catalogue/models/component/__init__.py

```diff
@@ -147,6 +148,55 @@ def __str__(self):
         return ""
 
 
+class Contact(models.Model):
+    # Contact Information
+    contact_person_name = models.CharField(
+        "Name",
+        help_text="Name des zuständigen Mitarbeiters",
+        max_length=100,
+    )
+    contact_address_zip = models.CharField(
+        "PLZ",
+        help_text="PLZ in der sich der Anbieter befindet",
+        max_length=100,
```

Comment:
max length 10? Also, for Germany integer, but we want to keep this compatible for incorporating other countries like the Netherlands.
 https://gist.github.com/jamesbar2/1c677c22df8f21e869cca7e439fc3f5b